### ISSUE TITLE
no blank line between usage and options when option is unknown

### ISSUE THREAD

**Issue by thorstenkampe** [TylerTemp/docpie#11]
Given below docstring, `docpie` does not print an empty line between usage and options:
```
"""
`SCRIPT` DESCRIPTION

Usage:
 SCRIPT [options]

Options:
 -h, --help   show help
"""
```
Output:
```
Unknown option: -n.

Usage:
 SCRIPT [options]
Options:
 -h, --help   show help
```

**Comment by TylerTemp** [comment]
related to #10

**Comment by thorstenkampe** [comment]
`Docopt` only prints usage - not options.

**Comment by TylerTemp** [comment]
fixed in `0.4.0` and released. Feel free to reopen if it still happens

**Comment by thorstenkampe** [comment]
Sorry, still not fixed. It's easy verifiable:
```
"""
`SCRIPT` DESCRIPTION

Usage:
 SCRIPT [options]

Options:
 -h, --help   show help
"""

import docpie

arguments = docpie.docpie(__doc__)
```
Could you please reopen the issue?

**Comment by TylerTemp** [comment]
fixed in 0.4.1

[online demo](http://docpie.comes.today/try/?doc=%60SCRIPT%60+DESCRIPTION%0D%0A%0D%0AUsage%3A%0D%0A+SCRIPT+%5Boptions%5D%0D%0A%0D%0AOptions%3A%0D%0A+-h%2C+--help+++show+help&argv=--unknow-option&run=on&help=on&stdopt=on&attachopt=on&attachvalue=on&auto2dashes=on)

Feel free to reopen if it still happens
